src/axl.c

```diff
@@ -573,9 +652,29 @@ int AXL_Dispatch (int id)
         AXL_ERR("Could not find transfer info for UID %d", id);
         return AXL_FAILURE;
     }
+    if (resume) {
+        switch (xstate) {
+        case AXL_XFER_STATE_NULL:
+            AXL_ERR("Error: Starting AXL_Resume() in state AXL_XFER_STATE_NULL\n");
```

Comment:
I think our AXL_ERR macro tacks on a error prefix string and a newline, so you can leave those off.
```suggestion
            AXL_ERR("Starting AXL_Resume() in state AXL_XFER_STATE_NULL");
```